src/consensus/params.h

```diff
@@ -14,6 +14,9 @@ namespace Consensus {
  */
 struct Params {
     uint256 hashGenesisBlock;
+
+    bool coinbaseMustBeProtected;
```

Comment:
Change to `fCoinbaseMustBeProtected` for consistency? I don't know why they picked `f` as the prefix (probably because `b` could imply bytes), but all consensus parameters are prefixed with their type.